agent/lib/docker/container.rb

```diff
@@ -34,6 +34,12 @@ def restart_policy
       self.host_config['RestartPolicy']
     end
 
+    # @return [String] Stack aware name of the containers service
+    def service_name_for_lb
```

Comment:
Note: Once we change the server `GridService#name_with_stack` to omit the `null-` prefix, then this is just `service_name`, because that's what it is in the agent RPC API?